cmd/user-agent/src/pages/chapi/wallet/didcomm/connections.js

```diff
@@ -20,20 +23,24 @@ export class AgentMediator {
 
     async connect(endpoint) {
         let invitation = await createInvitationFromRouter(endpoint)
-        let aries = this.aries
-        let conn = await aries.didexchange.receiveInvitation(invitation)
 
-        await this.waitFor(conn.connection_id, 'invited', function () {
-            return aries.didexchange.acceptInvitation({
-                id: conn.connection_id
-            })
+        let conn = await this.aries.outofband.acceptInvitation({
+            my_label: 'agent-default-label',
```

Comment:
may be a follow up on label, created https://github.com/trustbloc/edge-agent/issues/276